src/core.py

```diff
@@ -197,10 +197,9 @@ def setup_output(paths, template):
     ns.OUTPUT_SET = []
     ns.files_obj = {}
     ns.filesMap = {
-        "Results": paths.results_dir + "Results_" + TIME_NOW_HRS + ".csv",
-        "MultiMarked": paths.manual_dir + "MultiMarkedFiles_.csv",
-        "Errors": paths.manual_dir + "ErrorFiles_.csv",
-        "BadRollNos": paths.manual_dir + "BadRollNoFiles_.csv",
+        "Results": f"{paths.results_dir}Results_{TIME_NOW_HRS}.csv",
```

Comment:
`os.path.join(paths.results_dir, f"Results_{TIME_NOW_HRS}.csv")` would be a safer approach. 

For that, you would have to change `self.{results, manual, errors, multi_marked}_dir` too

Reply:
Maybe it'll be better to approve and merge this and make all `os.path` changes in a separate PR

Reply:
yep I'll add a comment for now.